pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -115,24 +116,24 @@ func getIgnitionConfig(assetsDir string, profile *performancev1alpha1.Performanc
 		},
 	}
 
+	// add script files under the node /usr/local/bin directory
+	mode := 0700
 	for _, script := range []string{hugepagesAllocation} {
-		content, err := ioutil.ReadFile(fmt.Sprintf("%s/scripts/%s.sh", assetsDir, script))
-		if err != nil {
+		src := fmt.Sprintf("%s/scripts/%s.sh", assetsDir, script)
```

Comment:
this should ne filepath.Join - but better fix in another PR

Reply:
done